content/en/bits_ai/managing_incidents.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Managing Incidents
+kind: guide
+disable_toc: false
+private: true
+is_beta: true
+---
+
+<div class="alert alert-warning">The following features are part of the <a href="https://www.datadoghq.com/product/incident-management/">Datadog Incident Management</a> product.</div>
+
+## Prerequisites
+
+- The "Mirror all incident Slack channel messages to its incident timeline in real-time" option must be turned on in **[Incident > Settings > Integrations][3] > Slack**. This enables Datadog to generate summaries and postmortems from Slack conversations.
+- The incident channel name must be prefixed with `#incident-`. This enables you to use the Bits AI chatbot in Slack (`@Datadog`) to ask questions about the incident or give updates on it.
+
+{{< img src="bits_ai/integration-settings.png" alt="Screenshot of Slack integration settings in Datadog" style="width:90%;">}}
+
+## Generate incident summaries and scheduled updates
+
+To generate an incident summary,
```

Comment:
When you join an incident channel in Slack that is connected to Datadog Incident Management, you will automatically receive a summary of the incident. A summary is only generated when a channel has 15 or more messages. The summary is only visible to you—and it does not persist across reloads, between desktop and mobile apps, or across sessions.